File: 02-Funcion/02-Go/HolaMundo.go
/*
	Programa que muestra el mensaje "Hola Mundo" en la pantalla
	Autor: Julián Esteban Gutiérrez Posada
	Fecha: Enero 2025
	Licencia: GNU GPL v3
*/

package main

import "fmt"

func main() {
	mostrarMensaje("Hola Mundo\n")
}

func mostrarMensaje(mensaje string) {
	fmt.Print(mensaje)
}

File: 02-Funcion/05-Haskell/HolaMundo.hs
{-
    Programa que muestra el mensaje "Hola Mundo" en la pantalla
    Autor: Julián Esteban Gutiérrez Posada
    Fecha: Enero 2025
    Licencia: GNU GPL v3
-}

main :: IO()
main = do
  mostrarMensaje "Hola Mundo\n"

mostrarMensaje :: String -> IO ()
mostrarMensaje mensaje = do
  putStr mensaje